tools/api-markdown-documenter/api-report/api-markdown-documenter.api.md

```diff
@@ -204,7 +204,7 @@ function createSummarySection(apiItem: ApiItem, config: ApiItemTransformationCon
 function createThrowsSection(apiItem: ApiItem, config: ApiItemTransformationConfiguration, headingText?: string): Section | undefined;
 
 // @public
-function createTypeParametersSection(typeParameters: readonly TypeParameter[], contextApiItem: ApiItem, config: ApiItemTransformationConfiguration): Section;
+function createTypeParametersSection(typeParameters: readonly TypeParameter[], contextApiItem: ApiItem, config: ApiItemTransformationConfiguration): Section | undefined;
```

Comment:
Note for reviewers: this change aligns the behavior of this function with the other section creation helpers we export.